### ISSUE TITLE
FormControlInput is not a constructor

### ISSUE THREAD

**Issue by geirman** [aichbauer/styled-bootstrap-components#14]
Firstly, I think this repo looks awesome, and I was excited to check it out. But unfortunately I ran into problems right away trying to create a form. 

Using NextJS, I created the following test component...

```
import { FormGroup, Label, FormControl } from 'styled-bootstrap-components'

export default props => (
  <>
    <h1>Test Page</h1>
    <form>
      <FormGroup>
        <Label>Email or Username</Label>
        <FormControl placeholder="Email or Username" />
      </FormGroup>
    </form>
  </>
)
```

But got this error...

```
FormControlInput is not a constructor

TypeError: FormControlInput is not a constructor
    at FormControl (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/styled-form-component/lib/components/FormControl.js:96:10)
    at processChild (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/react-dom/cjs/react-dom-server.node.development.js:2790:14)
    at resolve (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/react-dom/cjs/react-dom-server.node.development.js:2714:5)
    at ReactDOMServerRenderer.render (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/react-dom/cjs/react-dom-server.node.development.js:3098:22)
    at ReactDOMServerRenderer.read (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/react-dom/cjs/react-dom-server.node.development.js:3057:29)
    at renderToString (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/react-dom/cjs/react-dom-server.node.development.js:3524:27)
    at renderPage (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/next/dist/server/render.js:319:26)
    at Function.getInitialProps (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/.next/server/static/development/pages/_document.js:164:18)
    at _callee$ (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/next/dist/lib/utils.js:86:30)
    at tryCatch (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:5:24)
    at _next (/Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:27:9)
    at /Users/ChrisGeirman/dev/www/fq.next-yoga-prisma/client/node_modules/@babel/runtime-corejs2/helpers/asyncToGenerator.js:34:7
    at new Promise (<anonymous>)
```

To test, I verified that the error goes away if I replace `FormControl` with the following styled input...

```
const FormControl = styled.input``
```

Here's the versions I have installed...

```
    "styled-bootstrap-components": "^2.0.0",
    "styled-components": "^4.1.2",
    "styled-system": "^3.1.11",
```

**Comment by aichbauer** [comment]
Thank you 👍 

I think this problem is related to #12. I am right now working on a new release that should fix all this behavior with styled-components 4.x.x. Hopefully in the next few days.

**Comment by richardblondet** [comment]
Hello @aichbauer thanks for your amazing work. I just ran into this issue. What's the best way to move along? 
Should we downgrade `styled-components` ? 

Thanks!

**Comment by aichbauer** [comment]
@richardblondet thanks 😀 

Right now the only way to use this library completely is to use styled components v3.x.x but I am working on a new version hopefully releasing this week. The library fixed but I have to get some tests to pass. But this new version comes with complete new possibilities to customize this components to fit your needs.

**Comment by aichbauer** [comment]
closed by #15
